app/models/artist_version.rb

```diff
@@ -52,6 +45,13 @@ def search(params)
         q = q.where("is_banned = false")
       end
 
+      params[:order] ||= params.delete(:sort)
+      if params[:order] == "name"
+        q = q.order("name")
```

Comment:
This needs to be `q.order("name, id desc")`. Previously the `.order("id desc")` clause in the controller effectively did this, so removing that means it needs to be added here.